### DISCUSSION TITLE
react vant 社区支持 🎉

### DISCUSSION THREAD

**Discussion by 3lang3** [youzan/vant#9311] [Show and tell]
根据vant v3版本做了react版本的迁移，符合v3规范。

除业务组件和部分重型组件未搬迁，其余已经搬迁完毕，基本支持生产，公司内部项目落地，社区也有团队使用。

当然，此库正式版本还未发布，在此交流也是为了找到有相同需求的同学一起迭代更新，更好的反馈社区。

https://github.com/3lang3/react-vant

**Comment by 3lang3** [comment]
项目基本是结合react特性然后照搬v3的架构

- cli
- markdown
- fabir

**Comment by 3lang3** [comment]
我也看到社区其他同学在做react的支持，最终产出感觉都不太适合生产，或者转为不活跃状态了。

**Comment by chenjiahan** [comment]
非常棒~ 已经将链接添加到官方文档，期待更多维护 ^_^

**Comment by 3lang3** [comment]
谢谢了。
尽量跟上你们步伐，发到这里也是希望有更多同学参与进来😄

**Comment by 3lang3** [comment]
@chenjiahan 方便在vant-contrib.gitee.io上体现吗🤩

**Comment by chenjiahan** [comment]
![image](https://user-images.githubusercontent.com/7237365/130436418-b06a9b94-c324-4bb6-8d61-3977f30dccea.png)

已经有了喔

**Reply by 3lang3** [reply]
谢谢！
